packages/drivers/odsp-driver-definitions/api-report/odsp-driver-definitions.legacy.alpha.api.md

```diff
@@ -73,11 +73,15 @@ export interface IOdspErrorAugmentations {
 
 // @alpha (undocumented)
 export interface IOdspResolvedUrl extends IResolvedUrl, IOdspUrlParts {
+    // (undocumented)
+    appName?: string;
```

Comment:
Please add comments for this and context as well. You can take it from the constructor of the resolver where they are described.

Reply:
done